esipy/security.py

```diff
@@ -1,26 +1,33 @@
 # -*- encoding: utf-8 -*-
 from __future__ import absolute_import
 
-from .events import after_token_refresh
-from .exceptions import APIException
+import base64
+import logging
+import time
 
 from requests import Session
 from requests.utils import quote
-from six.moves.urllib.parse import urlparse
 
-import base64
-import logging
-import time
+from .events import AFTER_TOKEN_REFRESH
+from .exceptions import APIException
+
+try:
+    from six.moves.urllib.parse import urlparse
+except ImportError:
+    from urllib.parse import urlparse

```

Comment:
This is useless: you __need__ six for esipy, so it __can't__ do this exception. (if it does, you are doing something wrong). Also, this is the same import, just written differently as six mimics the structures of moved libs between py2/py3

Reply:
ill have to look over my system for that one, was saying it couldn't import six for some reason